src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -386,6 +386,11 @@ public async Task GetAsync_IncompleteData_ThrowsHttpRequestException(bool failDu
                 return;
             }
 
+            if (PlatformDetection.IsBrowser && failDuringHeaders)
+            {
+                return; // chrome browser doesn't validate this
```

Comment:
I think you can use `Assert.Skip("chrome browser doesn't validate this")` here.

Reply:
or `Assert.SkipWhen(PlatformDetection.IsBrowser && failDuringHeaders, "chrome browser doesn't validate this")`

Reply:
There is no such thing in XUnit, just NUnit